erdpy/cli_transactions.py

```diff
@@ -57,7 +61,11 @@ def create_transaction(args: Any):
         return
 
     try:
-        if args.send:
+        if args.with_result and args.send:
+            proxy = ElrondProxy(args.proxy)
+            response = proxy.send_transaction_and_wait_for_result(tx.to_dictionary(), args.timeout)
```

Comment:
Did not notice this before, but below, we have `tx.send(proxy)`. Here it's reversed: `proxy.send_transaction(tx)`. Can we add a method on `tx` so that it is consistent? Something like `tx.send_wait_result(proxy, timeout)`?

Reply:
Done.